blog/2020-12-18-How-is-office-culture-produced-maintained-transformed.md

```diff
@@ -0,0 +1,61 @@
+
+## Act I: Making culture out of nothing at all
+
+### How do you create culture?*
+
+It is not straightforward, and certainly “not something you have control over”, says Subra, who has been with Holmusk since 2015. Workplace culture is organic, and a function of the type of people hired into the company. Just like the personalities of the employees and co-workers, the personality of the company is not something that can be changed easily. Thus, I think the key in creating a good workplace culture lies in the very first step: hiring the right people.
+
+The right person at Holmusk is both work-oriented and people-oriented, and neither quality is solely sufficient. Employees have to care about the work they do while having camaraderie with the rest of the team. The hiring team “wasn’t interested in the best programmers or people who could write an app in a day, but people who can deliver a product together as a team”, adds Oishik, who has worked with Holmusk since starting here as an intern in 2019. Having people who can work well together and be friends makes life easier and work more productive and enjoyable. This is especially true in the case of meetings, which often take up a significant proportion of one's working hours, making it a draining task. Having meetings with people you enjoy being around can therefore have a big positive impact on the enjoyability of the work overall.
+
+### How do you hire the right people?
+
+It appears that the key to finding out if someone is the right fit lies in this sifting tray called the team lunch. This team lunch, which introduces the new candidate to 4-5 other representatives from various teams (e.g. UX, Software, Product, etc.) over a casual meal, happens during the hiring process to understand if the applicant fits well with the rest of the existing team. “When a new person joins, it is important for us to meet them physically to about them and what they want to learn and experience,” says Gracia, who has worked with Holmusk since 2018.
+
+### Does moving everything online change this?
+
+The pandemic has caused the Singapore government to place strict restrictions on group gatherings. Therefore, team lunches are now being held online. In a teleconference setting, Gracia believes it will be harder to understand her counterparts as well; “I don’t have the chance to understand their working habits, and I won’t be able to help you as effectively.” This sentiment is shared by Subra and Oishik too. Subra added that it used to be a purely social interaction, but now it slightly resembles a second interview. “It is definitely harder to do online”, Oishik recognises. Some of the drawbacks include missing out on visual cues and body language, “but it’s better than nothing, and is a good approximation.” It isn’t ideal, but it will do for now.
```

Comment:
@kwannoel, we can change the "their" -> "your" (though strictly speaking, you should always reach out to the person who is quoted).

Reply:
I believe it should be "help you" -> "help them" instead. Will check with Gracia.

Reply:
Gracia has rephrased it to:

**"Due to the limitation of physical interactions, we have to come up with creative ways to engage with our colleagues more effectively."**

Since the former sentence had a negative flair.